### DISCUSSION TITLE
Deepgram error Service Unavailable

### DISCUSSION THREAD

**Discussion by PetabitMan** [deepgram/community#142] [General help]
Receiving this error since this morning. 

>  DG: {"err_code":"Service Unavailable","err_msg":"Please try again later.","request_id":"---------"}

Anyone ideas why that might be?

**Comment by lukeocodes** [comment]
@PetabitMan seems like there was a momentary interuption in certain locations. Is this still occuring for you?

**Reply by PetabitMan** [reply]
No all fine now. Thank you.

**Comment by JoshJarabek7** [comment]
Happening to me right now as well /:
